src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -1043,6 +1046,137 @@ def print_fields(fields):
                 quoting=csv.QUOTE_NONE,
             )
 
+def extract_assay_expression(parser_obj):
+    """
+    Retrieve the selected data or generate SQL to retrieve the data from an expression assay in a dataset or cohort based on provided rules.
+    """
+
+    # Validating input combinations
+    parser_dict = vars(parser_obj)
+
+    schema = {
```

Comment:
Great! As this is a constant, let's make it one (capital letters, declared at the start of the file).